config.ru

```diff
@@ -1,5 +1,6 @@
 #Setup Load Path
-$LOAD_PATH << File.expand_path("../lib", __FILE__)
+$LOAD_PATH << File.expand_path("../lib", __FILE__) << File.expand_path("../../Apprenticeship-RubyTicTacToe/lib", __FILE__)
```

Comment:
Instead of that it would be nicer to do the following.

```
lib = File.expand_path("../lib", __FILE__) << File.expand_path("../../Apprenticeship-RubyTicTacToe/lib", __FILE__)
$:.unshift(lib)
```
For one it's cleaner in it's intent (_reveals what those files actually are_) and it's a little shorter without to much magic.

In case you are wondering `$:` is a shortcut for `$LOAD_PATH`.